docusaurus/docs/iOS/controllers/channels.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Channel Controllers
+---
+
+## ChannelListController
+
+`ChatChannelListController` allows you to observe a list of chat channels based on the provided query and to paginate channels.
+
+### ChannelList Delegate
+
+Classes that conform to the `ChatChannelListControllerDelegate` protocol will receive changes to the queried list of channels.
+
+```swift
+func controller(
```

Comment:
Consider changing `controller` to channelListController

Ignore this if there's a Swift convention underlying this just being called `controller`